pareto/operational_water_management/operational_produced_water_optimization_model.py

```diff
@@ -2594,6 +2605,29 @@ def TotalTreatmentCostRule(model):
 
     # model.TotalTreatmentCost.pprint()
 
+    # TODO: Figure out what happens with unused treated water. For now assume it gets disposed at the maximal disposal rate.
```

Comment:
update comment. For example: 
It is assumed that unused treated water leaves the water distribution network. The cost assumed includes disposal, additional treatment, and/or other cost of removing this water.